Assignment4/javipmontes-21A353/task06.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/100SqeqhtaxdAUCs75N5vzMjPfh7o7xAP
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This won't work in Python. You need to comment out line 12. The same applies for the other tasks